psi4/driver/driver_cbs.py

```diff
@@ -231,8 +243,8 @@ def scf_xtpl_helgaker_2(functionname: str, zLO: int, valueLO: float, zHI: int, v
         Higher zeta level. Should be equal to zLO + 1.
     valueHI
         Higher value used for extrapolation.
-    alpha
-        Overrides the default :math:`\alpha = 1.63`
+    alpha : float
+        Extrapolation parameter :math:`\alpha`.
 
     Returns
     -------
```

Comment:
I actually don't really know how to do this. Do I remove the type from the docstring completely, since the annotation is in the function declaration, or do I put Union[...] in there? @loriab

Reply:
returns processing is still less than satisfactory (c. spring 2021), and the same info needs to be repeated in the fn signature and in the docstring "Returns" section. unfortunately I can't find a good example to point you to that gets the type linked back to the docs correctly, much less a Union. so I think focus on getting the signature right and the docstring readable, and we'll just wait for the sphinx processing tech to catch up (and analyze warnings on a future sphinx run).